src/applications/caregivers/components/FormPages/FacilityConfirmation.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+
+const FacilityConfirmation = props => {
```

Comment:
Initial component to just render the selected facility. It will change with future tickets.

Reply:
nice incremental step!